ggml/src/ggml-cuda/ggml-cuda.cu

```diff
@@ -261,6 +261,11 @@ static ggml_cuda_device_info ggml_cuda_init() {
         GGML_LOG_INFO("  Device %d: %s, %s (0x%x), VMM: %s, Wave Size: %d\n",
                       id, prop.name, prop.gcnArchName, info.devices[id].cc & 0xffff,
                       device_vmm ? "yes" : "no", prop.warpSize);
+#elif defined(GGML_USE_MUSA)
+        // NOTE: MUSA will reserve some shared mem, and 24B should be enough,
```

Comment:
Sounds like a `TODO`.

Reply:
no it's not a `TODO`, it's just the current behavior of `musa` sdk.